### ISSUE TITLE
Common language snippets with multiple source files didn't work

### ISSUE THREAD

**Issue by afiqhamzah** [neoclide/coc-snippets#81]
## Neovim and coc.nvim status
![neovim_checkhealth](https://user-images.githubusercontent.com/26989296/70245878-c7fb3200-17b1-11ea-9a8c-7731c519b2d3.png)
![CocInfo](https://user-images.githubusercontent.com/26989296/70245868-c6316e80-17b1-11ea-8bb2-1f4e7ad21061.png?s=200&v=4)
## Vs Code Snippets 
[laravel 5 vs code snippets](https://github.com/onecentlin/laravel5-snippets-vscode)
## Issue
Snippets did not work
![laravel5-snippets_error](https://user-images.githubusercontent.com/26989296/70245875-c7629b80-17b1-11ea-9ea8-7060d588342e.png)

## Analysis and temporary solution
So I did make a comparison between package.json of laravel snippets and jest snippets that is used in this [tutorial](https://github.com/neoclide/coc.nvim/wiki/Using-snippets#using-vscode-snippet-from-extensions)
![comparison_package_jason](https://user-images.githubusercontent.com/26989296/70245871-c6ca0500-17b1-11ea-9d6c-8e5613ef60f7.png)
Unlike in jest snippets, all the language keys in laravel 5 snippets is php and are not unique, but the path properties are splitted into different files for different modules 
I'd then concatenated the json files under .config/coc/extensions/node_modules/laravel5-snippets/snippets using 
```bash
cat *.json > all.json
```
and edit the newly created file into valid json file, and modify the laravel5-snippet with the following changes
![modi](https://user-images.githubusercontent.com/26989296/70249081-d7c94500-17b6-11ea-89db-7904a0824f82.png)

And then finally it worked
![laravel5-snippets working](https://user-images.githubusercontent.com/26989296/70245872-c6ca0500-17b1-11ea-8612-0cdc6cea7fbe.png)

**Comment by joshukraine** [comment]
I believe that I am experiencing this same issue. I am trying to load [vue-vscode-snippets](https://github.com/sdras/vue-vscode-snippets) by Sarah Drasner. Her snippets are broken into multiple files, and in her [package.json file](https://github.com/sdras/vue-vscode-snippets/blob/master/package.json#L31) there are multiple entries for some filetypes. For example, these are the entries which are labeled "javascript":

```
    "snippets": [
      {...}
      {
        "language": "javascript",
        "path": "./snippets/vue-script.json"
      },
      {
        "language": "javascript",
        "path": "./snippets/vue-script-vuex.json"
      },
      {
        "language": "javascript",
        "path": "./snippets/vue-script-router.json"
      },
      {
        "language": "javascript",
        "path": "./snippets/nuxt-config.json"
      },
      {...}
    ]
```

I've extended my snippet config like so:

```
// coc-settings.json
"snippets.extends": {
  "vue": ["javascript"]
}
```

It appears that `coc-snippets` is only loading the first match, in this case `nuxt-config.json`. The others are ignored which means those snippets are unusable. Of course I can fork Sarah's repo and concatenate all the snippets into a single file, but it would be nice if `coc-snippets` could load multiple entries. @chemzqm maybe there is another workaround for this or something I've overlooked? Thanks!
